src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -0,0 +1,101 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    background-color: var(--color-black);
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  background-color: var(--color-black);
+  color: var(--color-white);
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding-bottom: 150px;
+  padding-top: 100px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 38px;
+    padding-top: 50px;
```

Comment:
I like this idea as well, although I think what we have is also OK.

In the future, when there's more browser support, it'd be great to use the [`gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) property with flexbox, since it'll help us add spacing on the insides of a flexbox container, rather than relying on the `:first-child`/`:last-child` overrides that we have to do. Support for this _just_ [landed in Chrome](https://web.dev/flexbox-gap/) a couple months ago, and Firefox has had it for a year, but Safari doesn't support it yet. Hopefully Apple adds support for it in Safari soon, since that means that we'll be able to make use of this, although we may still need to hold off from using it because not everyone updates their iPhones and Macs to get newer versions of Safari.

https://caniuse.com/flexbox-gap